.gitignore

```diff
@@ -8,3 +8,4 @@ iOS/.DS_Store
 node_modules/
 dist/
 **/config/*.env
+**/*/docker-compose.yml
```

Comment:
We should have this for the repo. I haven't added any changes for one yet but can you put it at the top level of the repository?